plugins/hls-refactor-plugin/test/Main.hs

```diff
@@ -1275,6 +1275,20 @@ extendImportTests = testGroup "extend import actions"
                     , "b :: A"
                     , "b = ConstructorFoo"
                     ])
+        , testSession "extend single line import in presence of extra perens" $ template
```

Comment:
```suggestion
        , testSession "extend single line import in presence of extra parens" $ template
```

If you want, you may just mark this test broken for ghc 92 :see_no_evil:

Reply:
On 9.2 the error doesn't contain any suggestion, which is why no code action is generated.
I tried for a bit to find an example that would trigger this on 9.2 but failed. I'll go with you suggestion.